<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplingProcessor</class>
 <widget class="QDialog" name="SamplingProcessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Cloud Sampling</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>POINT CLOUD PROCESSOR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="operationGroup">
     <property name="title">
      <string>Sampling Operation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="operationComboBox">
        <item>
         <property name="text">
          <string>Down Sample</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Up Sample</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sub Sample</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sampling</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="parametersGroup">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="downSampleLabel">
        <property name="text">
         <string>Down Sample Voxel Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="downSampleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="upSampleLabel">
        <property name="text">
         <string>Up Sample Multiplier:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="upSampleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="subSampleLabel">
        <property name="text">
         <string>Sub Sample Ratio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="subSampleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="sampleLabel">
        <property name="text">
         <string>Sample Count (Mesh to Point Cloud):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sampleSpinBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="exportPlyLabel">
          <property name="text">
           <string>Export to PLY:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="exportPlyCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>